source/_includes/feature.haml

```diff
@@ -3,16 +3,16 @@
   Feature pages are design documents that developers have created while collaborating on oVirt.
   %br
   %br
-  Most of them are 
+  Most of them are&nbsp;
   %strong
     outdated
   , but provide historical design context.
   %br
   %br
```

Comment:
This is not the right direction. Remove the "strong" tag instead. If a message need to use
"strong" the style should be set using css, not by this kind of ugly inline markup.

I would also remove this warning, it is not needed.

Reply:
Warning is there for a good reason, not going to be removed. the strong tag here is used in conformance to its definition:https://www.w3schools.com/tags/tag_strong.asp ; the fact it shows things in bold is merely a css configuraton, I agree.